libraries/rush-lib/src/cli/actions/PublishAction.ts

```diff
@@ -216,6 +216,9 @@ export class PublishAction extends BaseRushAction {
   protected async runAsync(): Promise<void> {
     PolicyValidator.validatePolicy(this.rushConfiguration, { bypassPolicy: false });
 
+    // FIXME: How to publish for packages in split workspace?
+    // Or, maybe we can not publish packages in split workspace (?)
```

Comment:
We might be able to skip this requirement, since in our monorepo, the "split" projects are the not-yet-migrated projects.  If the not-yet-migrated projects get published, hopefully they can use their old publishing job.  Then after they are migrated, they could get published by `rush publish`.   If we go that route, then Rush should report an error (rather than malfunctioning) if a pipeline tries to publish a split project.  A malfunctioning publishing job is very bad because it's not easy to unpublish a bad release.

Reply:
I tested `rush publish` for a split workspace project, it's actually working without any code changes... Do we need to block `rush publish` for "split" projects?

Reply:
Does PNPM correctly expand cross-workspace references in `workspace:*` specifiers in the **package.json** file that is being published? I wonder how it would resolve that if it cannot see the definition of the other workspace.